recognition/76-report/dice.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
```

Comment:
Code header block should be added for all scripts/source files, author info, file description etc.